pwiz_tools/Skyline/Model/Lib/Library.cs

```diff
@@ -1698,6 +1699,8 @@ public virtual float GetRankValue(PeptideRankId rankId)
         }
 
         public abstract IEnumerable<KeyValuePair<PeptideRankId, string>> RankValues { get; }
+        public abstract string Protein { get; } // Some .blib files provide a protein accession (or Molecule List Name for small molecules)

```

Comment:
I would probably have made this virtual and have the base class implementation return null, so that BiblioSpecLiteLibrary would be the only class that needed to override it.

Reply:
That has merit - less code, fewer changes - but I like the idea that if a new derived class is made the author has to at least think about this.